### ISSUE TITLE
Figure out Fetch interaction with html modules

### ISSUE THREAD

**Issue by annevk** [WICG/webcomponents#743]
Since fetches are happening from an unexpected place, need to make sure it's all good with respect to Fetch, CSP, et al and whether any special considerations apply.

**Comment by matthewp** [comment]
Fetch is happening from an unexpected place? What do you mean by that?

**Comment by domenic** [comment]
The context is cleverly hidden in the "modules" label.

**Comment by matthewp** [comment]
I'm still not sure I follow but I'll take a guess: is this about module scripts inside of a shadow root? Is that the unexpected place?

**Comment by annevk** [comment]
No, this is about what the behavior should be for elements that can load bits inside "HTML Modules".
